packages/azure-kusto-data/src/cloudSettings.ts

```diff
@@ -86,6 +86,17 @@ class CloudSettings {
         return urlString;
     }
 
+    private sanitizeUrlForLogging(kustoUri: string): string {
```

Comment:
Is there not a built in or existing function to do it already?
If not, is there a better place in the code to put it?